core/src/scene/sceneLoader.cpp

```diff
@@ -744,7 +744,12 @@ void SceneLoader::loadCameras(Node _cameras, Scene& _scene) {
             }
 
             if (Node vanishing = camera["vanishing_point"]) {
-                // TODO
+                if (vanishing.IsSequence() && vanishing.size() >= 2) {
```

Comment:
Probably not super important, but I'm guessing that JS doesn't check the array length is == 2.